src/library/scala/collection/parallel/ParIterableLike.scala

```diff
@@ -1284,7 +1282,7 @@ self: ParIterableLike[T, Repr, Sequential] =>
   extends Transformer[Combiner[(U, S), That], Zip[U, S, That]] {
     @volatile var result: Result = null
     def leaf(prev: Option[Result]) = result = pit.zip2combiner[U, S, That](othpit, pbf())
-    protected[this] def newSubtask(p: IterableSplitter[T]) = unsupported
+    protected[this] def newSubtask(p: IterableSplitter[T]) = throw new UnsupportedOperationException
```

Comment:
I'd have supported `unsupported()` with the side-effecty parens. Another edit in the PR adds parens to that effect. (oh, that was a good one.) I don't see how it affects dead-code, but it may or may not be desirable to narrow `newSubtask` to `Nothing`.